broker.go

```diff
@@ -82,6 +83,18 @@ func (b *Broker) Open(conf *Config) error {
 		b.conn = newBufConn(b.conn)
 
 		b.conf = conf
+
+		if conf.Net.SASL.Enable {
+			err = b.doSASLPlainAuth()
+			if err != nil {
+				Logger.Printf("SASL authentication with broker %s failed\n", b.addr)
```

Comment:
you log every case in `doSASLPlainAuth` *and* both cases here, that seems a bit much?